mergify_engine/web/__init__.py

```diff
@@ -329,28 +328,24 @@ async def marketplace_handler(
 
 
 @app.get(
```

Comment:
We should drop this method when the dashboard side have been updated to use the new one